src/PolymathAPI.ts

```diff
@@ -186,15 +181,15 @@ export class PolymathAPI {
     );
   }
 
-  public getTokens = async (params: GetTokensParams): Promise<PolyResponse> => {
-    assert.isNumber('amount', params.amount);
-    const address = !_.isUndefined(params.address) ? params.address : await this.getAccount();
-    assert.isETHAddressHex('address', address);
-
+  public getPolyTokens = async (params: GetTokensParams): Promise<PolyResponse> => {
     const networkId = await this.web3Wrapper.getNetworkIdAsync();
     if (networkId === 1) {
       throw new Error('Only for testnet');
     }
+    assert.isNumber('amount', params.amount);
+    const address = params.address !== undefined ? params.address : await this.getAccount();
+    assert.isETHAddressHex('address', address);
```

Comment:
you should validate if the eth address is not zero. `asserts` branch has a `isAddressNotZero` method that you can use here

Reply:
To avoid merge problems I'd say we should merge this one. Once done, I'll merge develop into `asserts` branch and then I can push a commit adding this validation.